### ISSUE TITLE
Upload the plugin into web store

### ISSUE THREAD

**Issue by gaocegege** [dyweb/colorful-contribution#1]
Now we have no doc or public page to show how to install the plugin, thus I think it is better to upload the plugin and add instruction doc.

**Comment by voldikss** [comment]
I think readme file is enough for GitHub users🧐.

**Comment by gaocegege** [comment]
Yeah README works well but there is no section about how to install the plugin.

**Comment by voldikss** [comment]
I think the best way is to host this extension on the Chrome Web Store. 
For Windows users,  installing extension outside the Store is not allowed..( [Chrome 33 Hosting Changes](https://developer.chrome.com/extensions/hosting_changes)

**Comment by gaocegege** [comment]
是的，我的意思是需要加一个文档关于如何安装它，无论是通过 web store 还是其他的

**Comment by gigaflw** [comment]
Added, thanks for mentioning! 👍 
But, I have yet to activate my chrome developer account, as well as google payment. So I'll probably leave aside uploading for now. Maybe after I've collected one or two more extensions to upload?

**Comment by at15** [comment]
now publishing using google group `dywebmos`, I don't know why it is showing `publishing in progress` https://stackoverflow.com/questions/25084330/chrome-extension-stalls-in-publishing-in-progress 

Also I need the following  @gigaflw 

- 1280x800 or 640x400 for screen shot
- icon

we may also need a google analytic account, https://github.com/dyweb/blog/issues/8

**Comment by gigaflw** [comment]
@at15 uploaded, 1 icon and 3 screenshots (640 x 400). see [releases](/dyweb/colorful-contribution/releases)
Thanks for uploading! 🌝 
Not quite clear about the google analytic things, how can I help ?

**Comment by at15** [comment]
published at https://chrome.google.com/webstore/detail/colorful-github-contribut/anigmebjfhogfilfobcddfjdjjaniclg

though I think we may need a better name

- there are `-` between names, if you search `github colorful contribution` you won't see the extension
- there are already several plugins with similar name and functionality ....

@gigaflw don't worry about the google analytic thing, @gaocegege and I will handle it

**Comment by gigaflw** [comment]
Also think we should have a better name. Blame me for that :neckbeard: 
Other plugins cannot handle icons, maybe we can have a name related to icons?

**Comment by voldikss** [comment]
@at15 It seems that the icon and screenshots were not uploaded successfully.
![default](https://user-images.githubusercontent.com/20282795/38200322-6fcc2dbe-36c6-11e8-8516-9938bb707a09.PNG)

**Comment by at15** [comment]
@clouduan I didn't upload it ... it requires a new release to bump the version number and also we may pick a new name  @gigaflw

**Comment by gigaflw** [comment]
Hmmm, I am not very good at naming....
How about Contribution Rug/Tapestry ? Or Contribution Decorator ?

**Comment by gaocegege** [comment]
I think we could remove `-` in the name

**Comment by gaocegege** [comment]
Why can't I find the plugin using the name colorful-github :thinking:

**Comment by gigaflw** [comment]
There is already an extension named 'Colorful Contributions for GitHub'. It can be too similar if we only remove the `-`

**Comment by at15** [comment]
so folks .... any idea about the name?

**Comment by at15** [comment]
@gigaflw @gaocegege ???

**Comment by gigaflw** [comment]
Emmmm, I'm working on a new feature which forms all small blocks into a large poster. After that, I believe this extension can be more distinguishable among all the colorful contribution extensions in extension store. I'll probably come up with a more specific name then. :smiley:

**Comment by gaocegege** [comment]
And please update the README, we already published into web store.

**Comment by gigaflw** [comment]
Hi! After this long time, I finally get the new version done.
Would you like to [take a look](https://github.com/dyweb/colorful-contribution/releases)? @gaocegege @at15 
Also @at15 , would you like to update the chrome store?

**Comment by gigaflw** [comment]
And, about the name. I'd like to change the name of the app on google store as well as this repo to **Github Tapestry**. What's your thoughts?

**Comment by at15** [comment]
@gigaflw you can ask @voldikss on how to upload to chrome store using travis https://github.com/dyweb/ACG-videos-detector/commit/0128f75fcb93195140516dffd9bfb5713ff1563f, I am not good at naming so I'll pass ... 

Though about the image used as background, is there any copyright issue?

**Comment by voldikss** [comment]
我当时参考的是这篇博文
https://circleci.com/blog/continuously-deploy-a-chrome-extension/

**Comment by gigaflw** [comment]
As far as I know, non-commercial use will not cause problem.
But I'm not sure. I will look into this problem a little more.
If you are talking about the images I used as presets, they are free to use.

**Comment by gigaflw** [comment]
Dyweb owns a chrome developer account right ?
Why not use that account and upload the extension as it was once done ?

**Comment by voldikss** [comment]
我刚刚上传到商店了，需要等一会才会生效。https://chrome.google.com/webstore/detail/anigmebjfhogfilfobcddfjdjjaniclg/

**Comment by at15** [comment]
Just came across this, shouldn't we update the README? ...
